cosv-frontend/src/main/kotlin/com/saveourtool/cosv/frontend/components/topbar/TopBarUserField.kt

```diff
@@ -86,12 +87,15 @@ val topBarUserField: FC<UserInfoAwareProps> = FC { props ->
                             }
                         }
                     }
-                    props.userInfo?.let { userInfo ->
+                    props.userInfo?.run {
```

Comment:
you have promised me to replace duplicated code in top bar. Do it pls

Reply:
Fixed it